docs/use/monitor_endpoints.md

```diff
@@ -0,0 +1,159 @@
+# Monitoring
+
+Percona Everest provides monitoring capabilities with PMM to maintain a reliable and secure database infrastructure.
+
+Here are some key values you'll get with monitoring of Percona Everest:
+
+- Monitoring helps track database availability and uptime
+- Monitoring provides insights into database performance
+- Monitoring helps detect and address issues proactively
+- Continuous monitoring helps maintain a stable and reliable database environment.
+
+
+## Prerequisites
+
+To use monitoring in Percona Everest, you should have a PMM instance up and running.
+
+For information on installing PMM, see the [documentation](https://docs.percona.com/percona-monitoring-and-management/setting-up/index.html).
+
+## Add monitoring from the UI
+
+!!! warning "Important"
+    When setting up DB cluster monitoring, using endpoints added through the user interface will result in a limited selection of metrics being sent to PMM. To receive the full range of metrics (including k8s data points), configure monitoring endpoints using the [Everest CLI](../install/installEverestCLI.md) `everestctl monitoring enable` command.
+
+To add monitoring in Percona Everest from the UI:
+{.power-number}
+
+1. From the Percona Everest Homepage, navigate to **Settings > Monitoring endpoints**. **Add monitoring endpoint** dialogue box opens.
+
+2. On the **Add monitoring endpoint screen**, enter a PMM instance name. In the **User** and **Password** field, enter the credentials received upon installing PMM. In the **Endpoint** field, enter the PMM URL.
+
+    ![!image](../images/everest_add_endpoint.png)
+
+
+## Edit monitoring from the UI
+
+To edit a monitoring endpoint from Percona Everest UI:
+{.power-number}
+
+1. From the Percona Everest Homepage, navigate to **Settings > Monitoring endpoints**.
+
+2. Click on the ellipsis (three dots) next to the endpoint you need to edit. The **Edit monitoring endpoint** dialogue box opens.
+
+     ![!image](../images/everest_endpoint_edit.png)
+
+3. Click **Add**.
+
+
+## Add monitoring from the CLI
+
+
+1. You can add monitoring in Percona Everest using **either** of the commands:
+
+    ```sh
+    everestctl monitoring enable
+    ```
+
+    Alternatively, navigate to the `percona-everest-cli` directory and run the following command:
+
+    ```sh
+    go run cmd/everest/main.go monitoring enable
+    ```
```

Comment:
I wouldn't mention running things from source, we want users to use the CLI releases.